src/datasources/work-orders/constants/WorkOrdersQueryBuilder.constants.ts

```diff
@@ -0,0 +1,170 @@
+import { QueryBuilderOperations } from "core/query-builder.constants";
+import { QBField } from "core/types";
+
+
+export enum WorkOrdersQueryBuilderFieldNames {
+    Name = 'name',
+    Description = 'description',
+    WorkOrderID = 'id',
+    State = 'state',
+    Type = 'type',
+    Workspace = 'workspace',
+    EarliestStartDate = 'earliestStartDate',
+    DueDate = 'dueDate',
+    CreatedAt = 'createdAt',
+    UpdatedAt = 'updatedAt',
+    AssignedTo = 'assignedTo',
+    RequestedBy = 'requestedBy',
+    CreatedBy = 'createdBy',
+    UpdatedBy = 'updatedBy',
+  }
+
+export const WorkOrdersQueryBuilderFields: Record<string, QBField> = {
+    NAME: {
+        label: 'Name',
+        dataField: WorkOrdersQueryBuilderFieldNames.Name,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name,
+            QueryBuilderOperations.CONTAINS.name,
+            QueryBuilderOperations.DOES_NOT_CONTAIN.name
+        ]
+    },
+    DESCRIPTION: {
+        label: 'Description',
+        dataField: WorkOrdersQueryBuilderFieldNames.Description,
+        filterOperations: [
+            QueryBuilderOperations.EQUALS.name,
+            QueryBuilderOperations.DOES_NOT_EQUAL.name
+        ]
+    },
+    WORK_ORDER_ID: {
+        label: 'Work Order ID',
```

Comment:
```suggestion
        label: 'Work order ID',
```

Can we match it to the query builder in the work orders grid?
![image](https://github.com/user-attachments/assets/0d06dd7f-b3bb-4389-a6bf-6bbac7e48773)

Reply:
updated